dts/arm/nordic/nrf9160ns.dtsi

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <arm/armv8-m.dtsi>
+#include <dt-bindings/i2c/i2c.h>
+#include <dt-bindings/gpio/gpio.h>
+
+/ {
+	cpus {
+		#address-cells = <1>;
+		#size-cells = <0>;
+
+		cpu@0 {
+			device_type = "cpu";
+			compatible = "arm,cortex-m33f";
+			reg = <0>;
+		};
+	};
+
+	sram0: memory@20000000 {
+		device_type = "memory";
+		compatible = "mmio-sram";
+	};
+
+	aliases {
+		rtc-1 = &rtc1;
+		clock = &clock;
+		power = &power;
+		nvmc = &nvmc;
+		uart-0 = &uart0;
+		uart-1 = &uart1;
+		uart-2 = &uart2;
+		uart-3 = &uart3;
+		adc-0 = &adc;
+		gpio-0 = &gpio0;
+		gpiote-0 = &gpiote; /* FIXME alias should be gpiote-1 */
+		i2c-0 = &i2c0;
+		i2c-1 = &i2c1;
+		i2c-2 = &i2c2;
+		i2c-3 = &i2c3;
+		spi-0 = &spi0;
+		spi-1 = &spi1;
+		spi-2 = &spi2;
+		spi-3 = &spi3;
+		wdt-0 = &wdt;
+		timer-0 = &timer0;
+		timer-1 = &timer1;
+		timer-2 = &timer2;
+	};
+
+	soc {
+		peripheral@40000000 {
+			#address-cells = <1>;
+			#size-cells = <1>;
+			ranges = <0x0 0x40000000 0x10000000>;
+
+			/* Include common nRF9160 peripheral description */
+			#include "nrf9160_common.dtsi"
+
+			/* Addditional Non-Secure peripherals */
+			gpiote: gpiote@40031000 {
+				compatible = "nordic,nrf-gpiote";
+				reg = <0x40031000 0x1000>;
```

Comment:
This should be `reg = <0x31000 0x1000>`, shouldn't it?

Reply:
Same comment as above :)